### DISCUSSION TITLE
Отключение плашки "перевести видео"

### DISCUSSION THREAD

**Discussion by Iwillbringdomination** [ilyhalight/voice-over-translation#511] [Q&A] [answered]
Можно ли как-нибудь отключать появление плашки "перевести видео" на определенных сайтах (например твиттер, вк) не залезая в код и не удаляя там что-либо?

**Comment by NullVerdict** [comment, marked as answer]
> Можно ли как-нибудь отключать появление плашки "перевести видео" на определенных сайтах (например твиттер, вк) не залезая в код и не удаляя там что-либо?

В настоящее время это невозможно, но вы можете попытаться реализовать эту функцию и сделать pull request, если хотите.
